project/views.py

```diff
@@ -15,7 +16,16 @@ class ProjectsListView(ListView):
     paginate_by = settings.PAGINATION
     context_object_name = 'projects'
     template_name = 'projects.html'
-    queryset = Project.objects.prefetch_related('members', 'technologies').all()
+    queryset = Project.objects.prefetch_related('members', 'technologies').all().order_by('-start_date')
+
+    def get(self, request, *args, **kwargs):
```

Comment:
you have the same functionality here, below and above. You can move them to mixin